kafka-ui-react-app/src/components/common/SmartTable/TableRow.tsx

```diff
@@ -61,12 +61,15 @@ export const TableRow = <T, TId extends IdType, OT = never>({
         if (!isColumnElement<T, TId>(child)) {
           return child;
         }
-        const { cell, field, width, className } = child.props;
+        const { cell, field, width, maxWidth, className } = child.props;
 
         const Cell = cell as React.FC<TableCellProps<T, TId, OT>> | undefined;
 
         return Cell ? (
-          <td className={className} style={{ width }}>
+          <td
+            className={className}
+            style={{ maxWidth, overflow: 'hidden', textOverflow: 'ellipsis' }}
```

Comment:
overflow: 'hidden', textOverflow: 'ellipsis' maybe those properties can be added by class rather than inline styling